radlab-ui/webapp/src/pages/api/deployments/[id]/status.ts

```diff
@@ -55,6 +55,9 @@ const getDeploymentStatus = async (
       return
     }
   })
+  const tfSteps = cloudBuild.steps.map((step: Record<string, any>) => {
```

Comment:
Instead of `Record<string, any>`, can you use the official type? https://cloud.google.com/build/docs/api/reference/rest/v1/projects.builds#Build.BuildStep

It should be something like `google.api.build.BuildStep`

Reply:
Created new type  according to official type

Reply:
Created new type  according to official type